tests/urls.py

```diff
@@ -13,7 +17,7 @@
     # shared among django-netjsonconfig components
     # keep the namespace argument unchanged
     url(r'^', include('django_netjsonconfig.urls', namespace='netjsonconfig')),
-    url(r'^', include('django_x509.urls', namespace='x509')),
+    url(r'^', include((django_x509_patterns, 'django_x509'), namespace='x509')),
```

Comment:
This change actually isn't needed right now since django-x509 was updated so I'll remove it - the pk to object_id change is necessary since django passes it as object_id in kwargs or something like that